### PR TITLE
Adds Sentry to the list of sponsors

### PR DESCRIPTION
**Pull request by tony-sull**
#### Description (required)

This adds Sentry to the list of sponsors in the main navigation sidebar

#### Related issues & labels (optional)

- Related astro.build [PR 879](https://github.com/withastro/astro.build/pull/879)

### CODE REVIEW

**Comment by sarah11918** [comment]
@tony-sull Is there a specific date for merging this? Does it wait for the launch day announcement?

**Comment by delucis** [comment]
+1 to Sarah’s question. And if we are waiting, this PR may not be so useful as we’ll be migrating to Starlight by then.

**Review by clearlyTHUYDOAN — APPROVED** [review]

**Comment by tony-sull** [comment]
@sarah11918 @delucis great question, I hadn't realized the Starlight migration was so close!

We have the green light to ship this update before the announcement, I'll be doing the astro.build merge shortly

**Comment by sarah11918** [comment]
@tony-sull @delucis It was just confirmed on Launch week call that yes, these logos can merge any time. So, we *can* merge here, and then we can carry it ourselves over to the migration, would you say, Chris?

**Comment by delucis** [comment]
Yup, that works!

File: src/components/LeftSidebar/Sponsors.astro [hunk 1]
```diff
@@ -26,6 +26,13 @@ import UIString from '../UIString.astro';
 			</g>
 		</svg>
 	</a>
+	<a href="https://sentry.io/" aria-label="Sentry">
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 44" width="110" height="24">
```
Comment (delucis) [inline src/components/LeftSidebar/Sponsors.astro:30]: LGTM! Just needs an `aria-hidden` on the SVG itself:

```suggestion
		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 44" width="110" height="24" aria-hidden="true">
```

Comment (tony-sull) [reply src/components/LeftSidebar/Sponsors.astro]: Great catch, I even saw that when copying it in and forgot about it when sitting back down to setup the PR 🤦

**Pull request merged**